hack/kube/overlays/dev-am/mysql-create-amss-location-job.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mysql-create-amss-location
+spec:
+  backoffLimit: 100
+  template:
+    spec:
+      serviceAccountName: sdps
+      restartPolicy: OnFailure
+      containers:
+        - name: create-locations
+          image: mysql:8.0
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: MYSQL_USER
+              valueFrom:
+                secretKeyRef:
+                  name: mysql-secret
+                  key: user
+            - name: MYSQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mysql-secret
+                  key: password
+            - name: AMSS_URL
+              valueFrom:
+                secretKeyRef:
+                  name: enduro-am-secret
+                  key: amss_url
+            - name: AMSS_USER
+              valueFrom:
+                secretKeyRef:
+                  name: enduro-am-secret
+                  key: amss_user
+            - name: AMSS_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: enduro-am-secret
+                  key: amss_api_key
+          command:
+            - sh
+            - -c
+            - >
+              LOC_AMSS='{"amss": {"url": "'$AMSS_URL'", "api_key": "'$AMSS_API_KEY'", "username": "'$AMSS_USER'"}}' &&
+              mysql -h mysql.enduro-sdps -u $MYSQL_USER -p$MYSQL_PASSWORD --execute "
+                INSERT IGNORE INTO enduro_storage.location (name, description, source, purpose, uuid, created_at, config)
+                VALUES ('amss', '', 'amss', 'aip_store', 'e0ed8b2a-8ae2-4546-b5d8-f0090919df04', NOW(), '$LOC_AMSS');
```

Comment:
Can we get the location UUID from the secret too? To make it match the one from the config directly in this environment.

Reply:
Yeah, good idea :+1:

Reply:
@jraddaoui I decided to make the default storage location configurable instead of having a separate AMSS location config variable.  Then for the AM dev environment I just override the config variable to use the AMSS location. Check out b13439c and see what you think.

I'll update the docs to match the changes in b13439c if we decide to go with it.

Reply:
Neat!